package.json

```diff
@@ -173,14 +175,15 @@
     "eslint": "^6.8.0",
     "eslint-plugin-ember": "^7.7.2",
     "eslint-plugin-node": "^11.0.0",
+    "express-pino-logger": "^4.0.0",
```

Comment:
don't change production code unrelated to your pull request.

Reply:
I think some of code we're change during the merge. I’ll correct that